packages/vbridge/src/components/Withdraw/Withdraw.tsx

```diff
@@ -134,13 +136,31 @@ const AddressAndInfoSection = styled.div`
 type WithdrawProps = {};
 
 export const Withdraw: React.FC<WithdrawProps> = () => {
```

Comment:
Can you explain why there's a lot of "deposit" wording in the Withdraw component? Since in the fucutre this will support transact -> withdraw, maybe there's a better name?